android-components/components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppShortcutManager.kt

```diff
@@ -236,15 +236,6 @@ class WebAppShortcutManager(
         return storage.recentManifestsCount(activeThresholdMs = activeThresholdMs)
     }
 
-    /**
-     * Updates the usedAt timestamp of the web app this url is associated with.
-     *
-     * @param manifest the manifest to update
-     */
-    suspend fun reportWebAppUsed(manifest: WebAppManifest): Unit? {
```

Comment:
Often this is done to surface the underlying API even if it's not used in Fenix or Focus. I think this would be considered a breaking change and would require a changelog entry for AC since this is affecting a public API.

My general recommendation is that we do nothing here because there may be external consumers of AC that are using this even if we aren't using it ourselves.

Reply:
Thanks @gabrielluong, good call out. Closing this PR and the related bug.

cc @MozillaNoah